src/newrelic/integration/test.go

```diff
@@ -315,6 +316,117 @@ func (t *Test) compareResponseHeaders() {
 	}
 }
 
+// Handling EXPECT_SPAN_EVENTS_LIKE is different than normal payload compare so
+// a different function is required
+func (t *Test) compareSpanEventsLike(harvest *newrelic.Harvest) {
+	// convert array of expected spans JSON to interface representation
+	var x2 interface{}
+
+	if nil == t.spanEventsLike {
+		return
+	}
+
+	if err := json.Unmarshal(t.spanEventsLike, &x2); nil != err {
+		t.Fatal(fmt.Errorf("unable to parse expected spans like json for fuzzy matching: %v", err))
```

Comment:
Isn't `return` needed here?

Reply:
Yes!  Thanks for catching this.
I misunderstood how the test failure mechanism worked but reviewing more of the existing code I see a return is required.

Fixed in commit [56511a1](https://github.com/newrelic/newrelic-php-agent/pull/717/commits/56511a1c7bb3aaa9a5fdb83077590f5dbe4721cf).